android/modules/ui/src/java/ti/modules/titanium/ui/widget/searchbar/TiUISearchBar.java

```diff
@@ -101,7 +101,7 @@ protected void onLayout(boolean changed, int left, int top, int right, int botto
 		promptText.setGravity(Gravity.CENTER_HORIZONTAL);
 		layout.addView(promptText, params);
 
-		params = new RelativeLayout.LayoutParams(LayoutParams.MATCH_PARENT, LayoutParams.MATCH_PARENT);
+		params = new RelativeLayout.LayoutParams(LayoutParams.MATCH_PARENT, LayoutParams.WRAP_CONTENT);
```

Comment:
This change looks like it will fix [TIMOB-25333](https://jira.appcelerator.org/browse/TIMOB-25333).

Reply:
I've confirmed it. The `SearchBar` height bug is fixed. I'll go ahead and reference this PR in the [TIMOB-25333](https://jira.appcelerator.org/browse/TIMOB-25333) ticket.